code/modules/mob/living/living.dm

```diff
@@ -845,11 +845,75 @@ default behaviour is:
 	..()
 
 //damage/heal the mob ears and adjust the deaf amount
-/mob/living/adjustEarDamage(var/damage, var/deaf)
+/mob/living/adjustEarDamage(var/damage, var/deaf, var/ringing = FALSE)
+	var/alreadydeaf = FALSE
+	if (ear_deaf)
+		alreadydeaf = TRUE
+
+	if (ishuman(src) && damage > 0)
```

Comment:
make a /mob/living/carbon/human subtype of the proc instead of checking if living is a human